cfgov/v1/models/base.py

```diff
@@ -21,6 +21,7 @@
     Orderable, Page, PageManager, PageQuerySet
 )
 from wagtail.wagtailimages.edit_handlers import ImageChooserPanel
+from wagtail.wagtailsearch import index
```

Comment:
I wonder if it matters here that we *don't* have the `wagtailsearch` app installed: https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/cfgov/settings/base.py#L49

At the very least, that means we can't use the manage.py subcommands provided by that app (this was intentional, since they conflict with haystack)

Reply:
besides the missing CLI tools, I think the only other problem might be, that these models don't get installed: https://github.com/wagtail/wagtail/blob/master/wagtail/search/models.py

They seem to exist for the sake of tracking query statistics. If you aren't getting any errors, maybe everything is fine?